content/200-concepts/100-components/02-prisma-client/index.mdx

```diff
@@ -1,7 +1,7 @@
 ---
 title: 'Prisma Client'
-metaTitle: 'Prisma Client (Reference)'
-metaDescription: 'Prisma Client is an auto-generated, type-safe query builder generated from the tables in your database.'
+metaTitle: 'Prisma Client'
```

Comment:
Is this the only page we have with "Prisma Client" as title?
https://www.prisma.io/client has a bit more at the end, so should be an ok overlap.

Reply:
Generally a domain should not have 2 pages with the same page title. As the `/client` one adds a bunch of stuff at the end we should be good. Otherwise it would have been and indication to get a bit more concrete in _this_ page title here in this page. Technically this is not "Prisma Client" but documentation of the Concept Prisma Client.